docs/EXTENDED-ARCHIVES.md

```diff
@@ -201,6 +212,15 @@ and `md5`.
 
 See the `.meta/checksum` file.
 
+##### `path (String)`
+
+The path to the image, relative to the root of the archive.
+
+##### `bmap (String)`
+
+The path a [`bmap`][bmap] file for the image, relative to the `.meta`
+directory.
```

Comment:
It feels like this conceptually belongs alongside the image, rather than in the `.meta` subdirectory? Just a gut-feeling.